integration-tests-en/scenarios/test_navigation/self_assessment_done_success.yml

```diff
@@ -5,5 +5,5 @@
   bot: test_navigation/explanations_assessment_done
 - user: continue
   bot: test_navigation/postal_code
-- user: test_navigation/postal_code/valid
-  bot: test_navigation/one_location_assessment_done
+# - user: test_navigation/postal_code/valid
+#   bot: test_navigation/one_location_assessment_done
```

Comment:
ML lost the insight of the ending "What else may I help you with?" but since message was replaced by one with the exact same meaning (May I help you with anything else?) and the same options, and that I did uncomment another part of a test, and that this PR was already enormous, I allowed myself to not add a ton of other stories to solve this...

Reply:
should add this comment in the file itself instead